packages/coil-client/src/queries/tip.ts

```diff
@@ -25,12 +25,19 @@ export interface TipInput {
 
 export async function tip(
   this: GraphQlClient,
-  token: string,
-  input: TipInput
+  token: string | TipInput,
+  input?: TipInput
```

Comment:
It is probably better here to just change the function signature to put input first

Reply:
The only "clients" for this lib are really in this repo. The package isn't published on npm or anything, so it's fine to break back-compat.